### ISSUE TITLE
🔗 ♿ 🐞 Aria Roles are missing from the nested (mega) Menu widget (Duplicate of #23875)

### ISSUE THREAD

**Issue by ivanverona1** [elementor/elementor#24998]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Hi everybody.

Is anyone good to trying help me about this problem? I found many other thread around the web but seem until now no one has found a way to solve this BUG. It’s seems to be an Elementor bug.

This is the github link about it: https://github.com/elementor/elementor/issues/23607

The bug seems to be still persistant after three years.

Does anyone wants provide to release a fix or something like that?

Reasoning in terms of Google lighthouse scores, with this problem you lose 5% of penalization in the accessibility score.

### Steps to reproduce

There are no particoulars steps to do to reproduce the bug. Just creating a menu in an elementor template. Testing the websiste in lighthouse you obtain a bad score in the accessibility.

### Expected behavior

Nothing to add here.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: nginx/1.21.4
	MySQL version: Ubuntu 20.04 v10.3.39
	PHP Version: 7.4.33
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 4000
	PHP Max Post Size: 8000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.3
	Site URL: https://onlygamers.it
	Home URL: https://onlygamers.it
	WP Multisite: No
	Max Upload Size: 2 GB
	Memory limit: 1024M
	Max Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: it-IT
	Timezone: Europe/Rome
	Debug Mode: Inactive

== Theme ==
	Name: Astra Child
	Version: 3.5.0.1680512088
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 4.6.4
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: it_IT
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36 Edg/121.0.0.0

== Active Plugins ==
	Admin Columns Pro
		Version: 6.4.4
		Author: AdminColumns.com

	Adminimize
		Version: 1.11.10
		Author: Frank Bültge

	Admin Menu Editor Pro
		Version: 2.23.3
		Author: Janis Elsts

	Advanced Custom Fields: Extended PRO
		Version: 0.8.9.5
		Author: ACF Extended

	Advanced Custom Fields PRO
		Version: 6.2.5
		Author: WP Engine

	Advanced Database Cleaner PRO
		Version: 3.2.7
		Author: Younes JFR.

	Advanced Editor Tools
		Version: 5.9.2
		Author: Automattic

	Advanced Post Queries
		Version: 1.1.1
		Author: Shabti Kaplan

	Asset CleanUp Pro: Page Speed Booster
		Version: 1.2.4.7
		Author: Gabe Livan

	Auto Image Attributes From Filename With Bulk Updater
		Version: 4.4
		Author: Arun Basil Lal

	Auto Image Attributes Pro
		Version: 4.4
		Author: Arun Basil Lal

	Bulk Move
		Version: 1.3.0
		Author: Sudar

	Child Theme Configurator
		Version: 2.6.5
		Author: Lilaea Media

	Classic Editor
		Version: 1.6.3
		Author: WordPress Contributors

	Copy Move posts
		Version: 1.6
		Author: Ujjaval Jani

	Custom Icons for Elementor
		Version: 0.3.1
		Author: Michael Bourne

	Custom Post Type UI
		Version: 1.15.1
		Author: WebDevStudios

	Database Cleaner: Clean, Optimize &amp; Repair (Pro)
		Version: 1.0.3
		Author: Jordy Meow

	Disable Admin Notices Individually
		Version: 1.3.2
		Author: Creative Motion <info@cm-wp.com>

	Disable Admin Notices premium
		Version: 1.1.0
		Author: CreativeMotion <wordpress.webraftic@gmail.com>

	Easy remove item menu
		Version: 1.0.6
		Author: Camilo

	Easy Theme and Plugin Upgrades
		Version: 2.0.2
		Author: Chris Jean

	Easy Updates Manager Premium
		Version: 9.0.17
		Author: Easy Updates Manager Team

	Ele Custom Skin
		Version: 3.1.7
		Author: Dudaster.com

	Ele Custom Skin Pro
		Version: 3.2.4
		Author: Dudaster.com

	Elementor
		Version: 3.19.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.19.0
		Author: Elementor.com

	FileBird Pro
		Version: 6.0.7
		Author: Ninja Team

	Filester - File Manager Pro
		Version: 1.8.2
		Author: Ninja Team

	Hide titles on hover
		Version: 0.0.5
		Author: Jose Mortellaro

	JetElements For Elementor
		Version: 2.6.15
		Author: Crocoblock

	JetEngine
		Version: 3.3.5
		Author: Crocoblock

	JetSearch
		Version: 3.2.2
		Author: Crocoblock

	JetSmartFilters
		Version: 3.3.1
		Author: Crocoblock

	Lazy Load Control For Elementor
		Version: 0.0.3
		Author: Jose Mortellaro

	Make Column Clickable Elementor
		Version: 1.4.0
		Author: Fernando Acosta

	Media Cleaner (Pro)
		Version: 6.7.0
		Author: Jordy Meow

	Media File Renamer: Rename Files (Manual, Auto, AI) (Pro)
		Version: 5.8.1
		Author: Jordy Meow

	New User Approve
		Version: 2.5.3
		Author: WPExpertsio

	Permalink Manager Pro
		Version: 2.4.1.6
		Author: Maciej Bis

	PowerPack Pro for Elementor
		Version: 2.10.10
		Author: Team IdeaBox - PowerPack Elements

	Premium Addons for Elementor
		Version: 4.10.18
		Author: Leap13

	Premium Addons PRO
		Version: 2.9.10
		Author: Leap13

	Smoother
		Version: 2.0.3
		Author: Merkulove

	TaxoPress Pro
		Version: 3.10.0
		Author: TaxoPress

	Unlimited Elementor Inner Sections By BoomDevs
		Version: 1.0.4
		Author: BoomDevs

	Unlimited Elements for Elementor (Premium)
		Version: 1.5.94
		Author: Unlimited Elements

	UpdraftPlus - Backup/Restore
		Version: 2.23.14.26
		Author: UpdraftPlus.Com, DavidAnderson

	Username Changer
		Version: 3.2.2
		Author: Widgit Team

	User Profile Picture
		Version: 2.6.1
		Author: Cozmoslabs

	User Role Editor Pro
		Version: 4.64.1
		Author: Vladimir Garagulia

	Video Thumbnailer for Elementor
		Version: 1.2.8
		Author: Engage Web, Nick Arkell

	Wordfence Security
		Version: 7.11.1
		Author: Wordfence

	Wordfence Security Activator
		Version: 1.2.0
		Author: mohamedhk2

	WPCode Pro
		Version: 2.1.8
		Author: WPCode

	WP Crontrol
		Version: 1.16.1
		Author: John Blackbourn & crontributors

	wpDiscuz
		Version: 7.6.14
		Author: gVectors Team

	WP OPcache
		Version: 4.2.0
		Author: nierdz

	WP Rocket
		Version: 3.15.8
		Author: WP Media

	WP Rollback
		Version: 2.0.6
		Author: WP Rollback

	WPS Hide Login
		Version: 1.9.13.2
		Author: WPServeur, NicolasKulka, wpformation

	Yoast SEO
		Version: 21.9.1
		Author: Team Yoast

	Yoast SEO Premium
		Version: 21.9
		Author: Team Yoast


== Must-Use Plugins ==
	Asset CleanUp Pro: Plugin Filtering
		Version: 1.0
		Author: Gabriel Livan


== Features ==
	Custom Fonts: 3
	Custom Icons: 0

== Integrations ==
	


== Esperimenti Elementor ==
	Caricamento delle risorse migliorato: Attiva
	Caricamento CSS migliorato: Attiva
	Icone dei font in linea: Inattivo per impostazione predefinita
	Punti di interruzione aggiuntivi: Attivo per impostazione predefinita
	admin_menu_rearrangement: Inattivo per impostazione predefinita
	Contenitore Flexbox: Attiva
	Aggiorna la libreria Swiper: Inattivo
	Contenitore Griglia: Inattivo per impostazione predefinita
	Default to New Theme Builder: Attivo per impostazione predefinita
	Barra unificata degli strumenti dell&#039;editor: Inattivo per impostazione predefinita
	Caricamento di Gutenberg ottimizzato: Inattivo
	Costruisci con l&#039;AI: Attivo per impostazione predefinita
	Landing Page: Attivo per impostazione predefinita
	Elementi annidati: Attiva
	Lazy Load immagini di sfondo: Inattivo per impostazione predefinita
	Caricamento immagine ottimizzato: Inattivo
	Guida di stile globale: Attivo per impostazione predefinita
	Page Transitions: Inattivo
	Notes: Inattivo
	Display Conditions: Inattivo per impostazione predefinita
	Form Submissions: Attivo per impostazione predefinita
	Scroll Snap: Inattivo
	Menu: Attiva
	Taxonomy Filter: Inattivo per impostazione predefinita

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by Daveden2** [comment]
Regarding the beta menu widget, there's an [ongoing discussion](https://github.com/elementor/elementor/issues/23875#event-11688713464) which you can follow and your detailed comments to. 

I feel there's no need to open another bug report

**Comment by nicholaszein** [comment]
Hello @ivanverona1!

Thank you for sharing your feedback.

Unfortunately, it wasn't clear from your report if this is a problem you are facing with the WordPress Menu (formerly known as the Nav Menu widget), or if you're referring to the new nested (mega) Menu widget.

After performing some tests with both the Nav Menu widget and the nested Menu widget, we couldn't reproduce what you are referring to here. Especially regarding the issue #23607. Which was already solved, and does not appear in the Lighthouse report.

Please, when submitting issues, be specific about what you are reporting, so we can understand it quickly, and test it ourselves.

Please clarify exactly what you are referring to so we can continue investigating this issue. Share screenshots, and if possible a screen recording of the issue. Point exactly where the problem is, and allow us to visit your website to see the problem _in-loco_. Right now when we try to visit your website we are blocked by your Cloudflare rules.

Kind regards

**Comment by ivanverona1** [comment]
Hi @nicholaszein, thanks to reply me. Can you please let me know from which country are you visiting my site? The widget that i'm using is the new nested (mega) of elementor. If you try to test my website on lighthouse you can cleary reproduce the error. Let me know for your country so i will provide to unlock it. Thanks.

**Comment by nicholaszein** [comment]
@ivanverona1 thanks for the quick reply!

We are visiting from Israel and Brazil.

**Comment by ivanverona1** [comment]
Ok i will unlock the site more soon as possibile. You can check whenever you want by tomorrow. Thanks to you  @nicholaszein

**Comment by nicholaszein** [comment]
Thanks @ivanverona1 🙏

**Comment by nicholaszein** [comment]
Hi @ivanverona1,

We're still not able to visit your website.

As this is part of the [ongoing discussion](https://github.com/elementor/elementor/issues/23875) @Daveden2 mentioned, I'm closing this issue as a duplicate.

Please, continue the conversation there, and let me know when you've opened your site for visitation.

Kind regards
